docs/source/platform/operation-registry.md

```diff
@@ -168,6 +169,23 @@ const server = new ApolloServer({
 });
 ```
 
+####5.1 Optionally, set the schema tag
+
+The plugin will automatically pickup the value of the environment variable `ENGINE_SCHEMA_TAG` and use that as the schema tag.  
```

Comment:
If users specify this env var, then our new changes will be breaking, since they'll pick up a new manifest. Let's make sure this is in the upgrade guide

Reply:
I will remove this for now to match changes in https://github.com/apollographql/apollo-platform-commercial/pull/125

Thanks for picking up on this 🙏